signInAndSignOut_feature/UI/PackagesViewController/PackagesViewController.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+class PackagesViewController: BaseViewController {
+    
+    // MARK: - IBOutlet -
+    @IBOutlet weak var tableViewPackages: UITableView!
+    
+    // MARK: - Variables -
+    var packagesArr = [Packages]()
+    
+    // MARK: - Life Cycle -
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateDataArr()
+    }
+    // MARK: - OVERRIDE FUNCTION -
+    override func viewWillAppear(_ animated: Bool){
+        self.navigationController?.navigationBar.isHidden = true
+    }
+    override func viewWillDisappear(_ animated: Bool) {
+        self.navigationController?.navigationBar.isHidden = false
+    }
+    // MARK: - IBAction -
+    
+    @IBAction func directTodashboardbtn(_ sender: Any) {
+        directTodashboard()
+    }
+    
+    // MARK: - Methods -
+    private func updateDataArr() {
+        tableViewPackages.register(UINib(nibName: "PackageCell", bundle: nil), forCellReuseIdentifier: "packagecell")
+        packagesArr.append(Packages(title: "21 Days Umrah Package", image: "packageImage"))
+        packagesArr.append(Packages(title: "41 Days Umrah Package", image: "packageImage"))
+        packagesArr.append(Packages(title: "61 Days Umrah Package", image: "packageImage"))
+        packagesArr.append(Packages(title: "81 Days Umrah Package", image: "packageImage"))
+    }
+    private func directTodashboard(){
+        self.popBack()
+    }
+}
+// MARK: - Extensions-
+extension PackagesViewController: UITableViewDataSource, UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return packagesArr.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableViewPackages.dequeueReusableCell(withIdentifier: "packagecell", for: indexPath) as! PackageCell
+        cell.updateData(for: packagesArr[indexPath.row])
+        cell.didSetDelegates(self, with: indexPath)
+        return cell
+    }
+    /*
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            didDelete(for: indexPath) // Pass the entire IndexPath
+        }
+    }*/
+}
+
+extension PackagesViewController: PackagesDelegates{
+    
+    func didEdit(for indexpath: IndexPath?) {
+        let alert = UIAlertController(title: "Edit Title", message: "Enter new title", preferredStyle: .alert)
```

Comment:
Why these strings are not called from StringConstants ?

Reply:
Updated